site/pt/blog/new-in-devtools-95/index.md

```diff
@@ -29,53 +28,61 @@ tags:
 
 
 <!-- ## New CSS length authoring tools {: #length } -->
-
+## Nova ferramenta de edição de comprimentos CSS {: #length }
 <!-- DevTools added an easier yet flexible way to update lengths in CSS! -->
-
+DevTools adicionou uma maneira mais fácil e flexível de atualizar comprimentos em CSS!
 <!-- In the **Styles** pane, look for any CSS property with length (e.g. `height`, `padding`). -->
-
+No painel **Styles**, procure qualquer propriedade CSS com comprimento (por exemplo, `height`,` padding`).
 <!-- Hover over the unit type, and notice the unit type is underlined. Click on it to select a unit type from the dropdown. -->
+Passe o mouse sobre o tipo de unidade e observe que o tipo de unidade está sublinhado. Clique nele para selecionar um tipo de unidade no menu suspenso.
```

Comment:
Question: is `menu suspenso` a well known translation for `dropdown menu`? This is the first time hearing this, but I may just not have heard this before.

Reply:
I've seen both just `dropdown` and `menu suspenso` in translations